docs/xdebug.md

```diff
@@ -1,19 +1,25 @@
 # Xdebug
 
-You can set the default value for the PHP_DEBUGGER int your `.env` file:
+You can set the default value for the PHP_DEBUGGER in your `.env` file:
 ````env
 PHP_DEBUGGER=xdebug
 XDEBUG_REMOTE_HOST=10.50.1.223
 ````
-Where `XDEBUG_REMOTE_HOST` is the ip of your IDE.
+
+`XDEBUG_REMOTE_HOST` is the ip-address of your IDE <br />
+Information on how to set up xdebug with PHPStorm is here: [Creating a PHP Debug Server](https://www.jetbrains.com/help/phpstorm/creating-a-php-debug-server-configuration.html)
+
+hint: if `PHP_DEBUGGER` is set to `xdebug`, then the xh-profiler won't profile anything 
```

Comment:
this info should be at the xh-profiler docs

Reply:
good point